### PR TITLE
[DE-Migration] LPS-125507 Avoid numeric fields being set to null

### PR DESCRIPTION
**Pull request by rodrigopaulino**
Related ticket: https://issues.liferay.com/browse/LPS-125507

### CODE REVIEW

**Comment by rodrigopaulino** [comment]
ci:test:sf

**Comment by rodrigopaulino** [comment]
ci:test:relevant

**Comment by rodrigopaulino** [comment]
These unique failures seem unrelated to the changes in this PR.

**Review by jeyvison — CHANGES_REQUESTED** [review]

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/DDMFormFieldTemplateContextFactory.java [hunk 1]
```diff
@@ -713,7 +713,9 @@ protected void setDDMFormFieldTemplateContextValueLocalizableValue(
 					ddmFormFieldValueAccessor.getValue(
 						ddmFormFieldValue, availableLocale);
 
-				if (!(ddmFormFieldValueAccessorValue instanceof BigDecimal)) {
+				if (!(ddmFormFieldValueAccessorValue instanceof BigDecimal) &&
+					!StringUtil.equals(ddmFormField.getType(), "numeric")) {
```
[the remark this answers is not in the document]
Comment (rodrigopaulino) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/DDMFormFieldTemplateContextFactory.java:717]: You got it

**Comment by rodrigopaulino** [comment]
@jeyvison https://github.com/liferay-data-engine/liferay-portal/pull/112

**Pull request closed without merging**